Resources/Audio/Effects/Footsteps/sources.txt

```diff
@@ -11,3 +11,5 @@ grass1.ogg and grass2.ogg are taken from https://freesound.org/people/D001447733
 grass3.ogg and grass4.ogg are taken from https://freesound.org/people/jevans27/sounds/563529/ by jevans27 licensed under CC-0 and split/modified by mirrorcult
 
 snowstep1.ogg, snowstep2.ogg, snowstep3.ogg, and snowstep4.ogg are taken from https://freesound.org/people/alexo400/sounds/543385/ by alexo400 licensed under CC-0 and split by EmoGarbage404
+
+For fisherboots1.ogg through fisherboots12.ogg - original sound from Pixabay at https://pixabay.com/sound-effects/rubber-squeak-type-07-192-mono-27378/ under CC0. Modified for use by The Best Nerd.
```

Comment:
Use attributions.yml

Also this isn't CC0. Their license says it only applies to works prior to 2019 and this looks like it's from 2023.

Reply:
Fixing this in the next commit.